3-docker-compose/README.md

```diff
@@ -2,7 +2,7 @@
 
 While docker allows you to easily build and link different apps with different environments and runtimes, doing so with more than just two or three containers quickly becomes tedious. `docker-compose` allows you to work with complex container systems very easily.
 
-For working with `docker-compose`, you will need a `docker-compose.yml` file. Check out the one under the `gvilarino/docker-testing` repo.
+For working with `docker-compose` you will need to [install it](https://docs.docker.com/compose/install/) depending on your case and you will need a `docker-compose.yml` file. Check out the one under the `gvilarino/docker-testing` repo.
```

Comment:
I'd do something like:

First check if you have `docker-compose` installed (chances are that you have) by doing:

```
docker-compose --version
```

If that fails, [get `docker-compose` for you specific setup](https://docs.docker.com/compose/install/)